src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -53,25 +53,11 @@ public function showIfNotAuthorizedDepartementAction(): Response
     public function showAction(Request $request, CitizenProject $citizenProject, CitizenProjectManager $citizenProjectManager): Response
```

Comment:
The `$request` variable and the `POST` method annotation are useful ?

Reply:
No